blackbox/docs/appendices/release-notes/unreleased.rst

```diff
@@ -122,6 +123,10 @@ Breaking Changes
 Deprecations
 ============
 
+- Removed deprecated ``nGram``, ``edgeNGram`` token filter and ``htmlStrip``
```

Comment:
Sorry but please move to ``Changes`` as we do not deprecate but remove them now.